### PR TITLE
Fix duplicated relu and wrong prefix

### PR DESCRIPTION
**Pull request by hetong007**
The PR intends to fix:

https://github.com/dmlc/gluon-cv/issues/408
https://github.com/dmlc/gluon-cv/issues/409

### CODE REVIEW

**Comment by hetong007** [comment]
@zhanghang1989 @zhreshold please review to avoid any potential impact on detection/segmentation models.

**Comment by mli** [comment]
Job PR-415-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-415/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-415-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-415/6/index.html 
Code coverage of this PR:  vs. Master:

File: gluoncv/model_zoo/resnetv1b.py [hunk 1]
```diff
@@ -81,11 +84,13 @@ def hybrid_forward(self, F, x):
 
         out = self.conv1(x)
         out = self.bn1(out)
-        out = self.relu(out)
+        out = F.Activation(out, act_type='relu')
```
Comment (Angzz) [inline gluoncv/model_zoo/resnetv1b.py]: why use two continuous reliable here？

Comment (hetong007) [reply gluoncv/model_zoo/resnetv1b.py]: Good catch! Patching in #416 .

**Pull request merged**